src/Kreta/TaskManager/Domain/Model/Organization/Organization.php

```diff
@@ -46,4 +55,34 @@ public function slug() : OrganizationSlug
     {
         return $this->slug;
     }
+
+    public function owners() : array
```

Comment:
This getter should be return a OwnerCollection itself, because we'll work with the collection inside domain not with a simple array.